pkg/karmadactl/karmadactl.go

```diff
@@ -96,6 +97,7 @@ func NewKarmadaCtlCommand(cmdUse, parentCommand string) *cobra.Command {
 			Commands: []*cobra.Command{
 				explain.NewCmdExplain(f, parentCommand, ioStreams),
 				get.NewCmdGet(f, parentCommand, ioStreams),
+				config.NewCmdConfig(parentCommand),
```

Comment:
The `config` command serves the `init` command, so it would be better to place it in the "Cluster Registration Commands:" group.